source/content/guides/drupal-9/drupal-9-hosted-createbt/11-deploy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Migrate a Site That Was Created with Build Tools to Drupal 9
+subtitle: Deploy
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-createbt/deploy
+anchorid: deploy
+editpath: drupal-9/drupal-9-hosted-createbt/11-deploy.md
+---
+<Partial file="drupal-9/deploy-using-relaunch.md" />
```

Comment:
Relaunch is definitely not needed here. Once everything is done and ready in the master branch you should just deploy to test and then live and that's all!

Reply:
Updated.